src/Benchmarks/Startup.cs

```diff
@@ -13,6 +14,7 @@ public class Startup
         public void ConfigureServices(IServiceCollection services)
         {
             services.AddMvc();
+            services.AddSingleton(typeof(IHttpContextAccessor), typeof(InertHttpContextAccessor));
```

Comment:
Is off in a non-nuget benchmarking project; rather than a main branch.

Maybe should add a big WARNING comment? Not sure of phrasing though... 

Probably good for webapi based stuff; inter-machine comms; not so good for user stuff where you need to login and have html rendered.

Will get some numbers together...